Hospital-System/HospitalSystem.BLL/Extensions/LoggerExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using HospitalSystem.DAL.Entities.Base;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Logging;
+
+namespace HospitalSystem.BLL.Extensions;
+
+public static class LoggerExtensions
+{
+    public static void LogIdentityErrors<T>(this ILogger<T> logger, User user, IdentityResult result)
```

Comment:
Are there any real sense to implement extensions method for a one-time logic?

This method used only once, and probably it makes sense to declare that as an internal method at the `UserService`.
![image](https://github.com/doomer2004/Hospital-system./assets/67170413/8b2bd786-b3b1-4b89-aa9f-cec976b77236)